packages/web/src/app/AppProviders.tsx

```diff
@@ -55,37 +57,39 @@ export const AppProviders = ({ children }: AppProvidersProps) => {
   })
 
   return (
-    <QueryClientProvider client={queryClient}>
-      <ReduxProvider store={store}>
-        <Router history={storeHistory}>
-          <CompatRouter>
-            <LastLocationProvider>
-              <RouterContextProvider>
-                <HeaderContextProvider>
-                  <NavProvider>
-                    <ScrollProvider>
-                      <ToastContextProvider>
-                        <AppContextProvider>
-                          <AudiusQueryProvider>
-                            <MainContentContextProvider>
-                              <ThemeProvider>
-                                <SvgGradientProvider>
-                                  {children}
-                                </SvgGradientProvider>
-                              </ThemeProvider>
-                            </MainContentContextProvider>
-                          </AudiusQueryProvider>
-                        </AppContextProvider>
-                      </ToastContextProvider>
-                    </ScrollProvider>
-                  </NavProvider>
-                </HeaderContextProvider>
-              </RouterContextProvider>
-            </LastLocationProvider>
-          </CompatRouter>
-        </Router>
-      </ReduxProvider>
-      <ReactQueryDevtools />
-    </QueryClientProvider>
+    <WagmiProvider config={wagmiConfig}>
```

Comment:
_we think in functional programming_

![image](https://github.com/user-attachments/assets/667fcd11-ce8d-442e-ad46-7cb3c0f74f73)